api/create_materialized_view.md

```diff
@@ -45,8 +45,14 @@ parameter. For more information, see
 
 Continuous aggregates have some limitations of what types of queries they can
 support, described in more length below. For example, the `FROM` clause must
-provide only one hypertable or underlying continuous aggregate, and joins, CTEs, or subqueries are not
-supported. The `GROUP BY` clause must include a time bucket on the underlying
+provide only one hypertable or underlying continuous aggregate, CTEs, or subqueries are not
+supported. The `FROM` clause now supports JOINS but it has to adhere to the following requirements,
+1. The JOIN can be between only two tables, where one is a hypertable (and NOT a continuous aggregate itself)
+   and other table is a normal PostgreSQL table
```

Comment:
Should we call those `normal` or `regular` tables?

What's your opinion @Loquacity @RafiaSabih?

Reply:
either is fine by me, no preferences there.

Reply:
We use "standard PostgreSQL": https://docs.timescale.com/timescaledb/latest/contribute-to-docs/#v